index.html

```diff
@@ -61,6 +61,277 @@ <h1>i'm Adan <br>
+                    <li class="images">CANOPY</li>
+                    <li id="fade">Back End Dev</li>
+                    <li id="fade">2015</li>
+                </ul>
+
+                <p class="brief">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <div class="languages">
+                    <ul class="html-css">
+                        <li>
+                            <a href=""><img src="./images/html.png" class="img2" alt=""></a>
+                        </li>
+                        <li>
+                            <a href="#index.html"><img src="./images/Tag.png" class="img2" alt=""></a>
+
+                        </li>
+                        <li>
+                            <a href="#index.html"><img src="./images/Tag (1).png" class="img2" alt=""></a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="see-projects">
+                    <button class="see">See Projects</button>
+                </div>
+
+            </div>
+
+
+
+            </div>
+            </div>
+        </section>
+
+        <section class="about-myself">
+            <div class="Portfolio">
+                <div class="intro">
+                    <h1>About
+                        <br>
+                        Myself</h1>
+                </div>
+                <div class="about">
+                    <p class="me">
+                        I’m a software developer! I can help you build a product , feature or website Look through some
+                        of
+                        my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate
+                        to contact me.</p>
+                </div>
+
+                <div class="connect">
+                    <a class="btn" href="#index.html">LET'S CONNECT</a>
+                </div>
+
+                <ul class="images">
+                    <li class="column">
+
+                        <a href=""><img src="images/twitter.jpg" alt="twitter"></a>
+                    </li>
+                    <li class="column">
+
+                        <a href=""><img src="./images/github.jpg" alt="github"></a>
+                    </li>
+                    <li class="column">
+                        <a href=""><img src="./images/chil.jpg" alt="chill"></a>
+                    </li>
+                    <li class="column">
+                        <a href=""><img src="./images/M.jpg" alt="m"></a>
+                    </li>
+                    <li class="column">
+                        <a href=""> <img src="./images/i.png" alt="i">
+                            <img src="./images/n.png" alt="m"></a>
+                    </li>
+
+
+                </ul>
+                <div class="get">
+                    <button class="resume">Get my resume</button>
```

Comment:
- [ ] Nice implementation of The `button` tag and please follow the button styling: Active, Hover and Pressed [(see design)](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)